tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-step-selector.html

```diff
@@ -0,0 +1,172 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-slider/paper-slider.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-card-heading/tf-card-heading.html">
+<link rel="import" href="../tf-color-scale/tf-color-scale.html">
+<link rel="import" href="../tf-imports/lodash.html">
+
+<!--
+  Renders a UI for selecting a step.
+-->
+<dom-module id="tf-pr-curve-step-selector">
+  <template>
+    <template is="dom-repeat" items="[[runs]]" as="run">
+      <div class="run-widget">
+        <div class="run-display-container">
+          <div class="run-color-box" style="background:[[_computeColorForRun(run)]];"></div>
+          <div class="run-text">
+            [[run]]
+          </div>
+        </div>
+        <div class="step-display-container">
+          step <span class="step-text">[[_computeStepTextForRun(runToStep, run)]]</span>
+        </div>
+        <paper-slider
+            id="slider-[[run]]"
```

Comment:
Whoops, my mistake—you need `data-run$="[[run]]"` to bind to the attribute. (I think `getAttribute("data-run")` should also work; if you prefer that, it's fine with me.)